src/index.html

```diff
@@ -3,11 +3,304 @@
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <a href="#" class="navigation__logo">
+          <img
+          class="navigation__logo-image"
+          src="./images/logo-mobile.svg"
+          alt="MyBiKE logo"
+          >
+        </a>
+
+        <a href="tel:+1 234 555-55-55" class="navigation__phone">
+          <span class="navigation__phone-number">+1 234 555-55-55</span>
+
+          <img
+          class="navigation__phone-icon"
+          src="./images/phone.svg"
+          alt="Phone icon"
+          >
+        </a>
+
+        <a href="#menu" class="navigation__burger"></a>
+      </nav>
+      <h1 class="title header__title">Take the Streets</h1>
+
+      <div class="menu" id="menu">
+        <nav class="navigation menu__navigation">
+          <a href="#" class="logo navigation__logo">
+            <img
+            class="logo__image"
+            src="./images/logo-mobile-dark.svg"
+            alt="MyBiKE logo"
+            >
+          </a>
+
+          <a href="#home" class="close navigation__item">
+            <img
+            class="close__icon"
+            src="./images/burger-close.svg"
+            alt="Menu-burger icon"
+            >
+          </a>
+        </nav>
+
+        <nav class="links menu__links">
+          <ul class="links__list">
+            <li class="links__item"><a href="#home" class="links__link">Home</a></li>
+            <li class="links__item"><a href="#about-us" class="links__link">About Us</a></li>
+            <li class="links__item"><a href="#compare-bikes" class="links__link">Compare Bikes</a></li>
+            <li class="links__item"><a href="#details" class="links__link">Details</a></li>
+            <li class="links__item"><a href="#contact-us" class="links__link">Contacts</a></li>
+          </ul>
+        </nav>
+
+        <span class="menu__paragraph">+1 234 5555-55-55</span>
+        <a href="#" class="menu__call-link">Book a test ride</a>
+      </div>
+    </header>
+
+
+
+    <main class="main" id="about-us">
+      <h2 class="title title--secondary main__title">
+        Move Free
+      </h2>
+
+      <p class="main__paragraph">
+        Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+      </p>
+
+      <section class="section main__section" id="compare-bikes">
+        <h3 class="title title--section section__title">
+          Compare bikes
+        </h3>
+
+        <article class="product-card section__product-card">
+          <img
+          class="product-card__image section__product-card--image"
+          src="./images/picture-1.jpg"
+          alt="Sporty 4 bike model image"
+          >
+
+          <h4 class="product-card__title section__product-card--title">Sporty 4</h4>
```

Comment:
There should be an empty line between this h4 and p, same for the rest of these articles